documentation/api/io.strimzi.api.kafka.model.connect.build.Build.adoc

```diff
@@ -187,3 +187,37 @@ spec:
 <1> (Required) Type of artifact.
 <2> (Required) URL from which the archive is downloaded.
 <3> (Optional) SHA-512 checksum to verify the artifact.
+
+.Using `other` artifacts
+
+`other` artifacts represent any kind of file which is downloaded and added to a container image.
+A file name which will be used in the resulting container image can be also specified using the `fileName` field.
```

Comment:
```suggestion
If you want to use a specific name for the artifact in the resulting container image, use the `fileName` field.
```

Reply:
Maybe add what happens if you don't add one?
_If the fileName is not specified, the file is named based on the URL hash._